schemas/github-pull-request-message.yml

```diff
@@ -0,0 +1,39 @@
+id: http://schemas.taskcluster.net/github/v1/github-pull-request-message.json#
+$schema:  http://json-schema.org/draft-04/schema#
+title:        "GitHub Pull Request Message"
+description: |
+  Message reporting that a GitHub pull request has occurred
+type:         object
+properties:
+  version:    {$const: message-version}
+  organization:
+    description: |
+      The GitHub `organization` which had an event.
+    type:         string
+    minLength:    {$const: identifier-min-length}
+    maxLength:    {$const: identifier-max-length}
+    pattern:      {$const: identifier-pattern}
+  repository:
+    description: |
+      The GitHub `repository` which had an event.
+    type:         string
+    minLength:    {$const: identifier-min-length}
+    maxLength:    {$const: identifier-max-length}
+    pattern:      {$const: identifier-pattern}
```

Comment:
I updated the original comment instead of adding another (I should sleep, haha!) update read:

So, the good news is, except for periods, github appears to be equally strict. I couldn't find docs either; but tested it out: http://imgur.com/VZldLOd

I'm going to leave the standard tc way + "%", but also add a little sanitizer. I'll write this in the summary. Let me know if you'd prefer another way.